test/errhandling/ferguson/output-format.chpl

```diff
@@ -9,7 +9,7 @@ class MyError : Error {
   }
 }
 proc throwit() throws {
-  var e = new MyError("test error");
+  var e = new unmanaged MyError("test error");
```

Comment:
The error handling (compiler-added) code assumes its responsible for deleting a thrown error. In the future though I hope it will generalize this stuff with `owned`. Right now `unmanaged` and `borrowed` are still close enough that you can delete a borrow - which is why it doesn't leak if you throw a borrow. But it should be an error.

Reply:
I made issue #9602 (mainly as a placeholder) to track that it shouldn't work in the future to throw a borrowed.